Userland/Libraries/LibGfx/Font/TrueType/Font.cpp

```diff
@@ -489,9 +489,9 @@ ErrorOr<NonnullRefPtr<Font>> Font::try_load_from_offset(ReadonlyBytes buffer, u3
         return Error::from_string_literal("Could not load Glyf");
     auto glyf = Glyf(opt_glyf_slice.value());
 
-    if (!opt_os2_slice.has_value())
-        return Error::from_string_literal("Could not load OS/2");
-    auto os2 = OS2(opt_os2_slice.value());
+    OS2 os2(TRY(ByteBuffer::create_zeroed(78)));
```

Comment:
Can this 78 be a `sizeof(...)` instead? Or failing that, given a name? It's not clear where it comes from.

Reply:
Sure, I added it as part of the OS2 class now.